client/src/modules/admin/components/admin-remove.component.js

```diff
@@ -0,0 +1,43 @@
+import Modal from 'react-bootstrap/Modal';
+import { toast } from 'react-toastify';
+
+import Button from '../../core/components/button.component';
+import { useDispatch } from 'react-redux';
```

Comment:
This is a node module import! this imports will go to the top of the import list